utils/bin/poller/src/main.rs

```diff
@@ -0,0 +1,301 @@
+use cow_amm_common::{
+    rpc_url, ComposableCoW,
+    ConstantProductHelper::{self, ConstantProductHelperErrors},
+    GPv2Settlement,
+    IConstantProductHelper::{self, dReturn},
+    IMulticall3, IPriceOracle, LegacyConstantProduct, LegacyTradingParams, Order, IERC1271,
+};
+use std::collections::HashMap;
+
+use std::str::FromStr;
+
+use alloy::{
+    contract::Error,
+    hex,
+    primitives::{Address, Bytes, FixedBytes, U256},
+    providers::{Provider, ProviderBuilder},
+    rpc::types::state::AccountOverride,
+    sol_types::{
+        decode_revert_reason, eip712_domain, SolCall, SolEvent, SolInterface, SolStruct, SolValue,
+    },
+    transports::TransportError,
+};
+
+#[tokio::main]
+async fn main() -> eyre::Result<()> {
+    const SETTLEMENT: &str = "0x9008D19f58AAbD9eD0D60971565AA8510560ab41";
+    const MUTLICALL3: &str = "0xcA11bde05977b3631167028862bE2a173976CA11";
```

Comment:
These are only ever used after `.parse()`, so I'd just do the parsing and unwrapping here.